aries-site/src/examples/templates/persistent-navigation/MinimalSidebarExample.js

```diff
@@ -0,0 +1,328 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Avatar,
+  Box,
+  Button,
+  Drop,
+  Grid,
+  Header,
+  Heading,
+  List,
+  Nav,
+  ResponsiveContext,
+  Sidebar,
+  Text,
+} from 'grommet';
+import {
+  Chat,
+  Cli,
+  Clock,
+  Configure,
+  Location,
+  Hpe,
+  Projects,
+  Splits,
+  StatusInfoSmall,
+  StatusUnknown,
+} from 'grommet-icons';
+
+const pages = [
+  {
+    name: 'Focus',
+    icon: <StatusInfoSmall />,
+    content: 'panes',
+  },
+  {
+    name: 'Services',
+    icon: <Projects />,
+    content: 'cards',
+  },
+  {
+    name: 'Glances',
+    icon: <Clock />,
+    content: 'cards',
+  },
+  {
+    name: 'Flows',
+    icon: <Splits />,
+    content: 'list',
+  },
+  {
+    name: 'Locations',
+    icon: <Location />,
+    content: 'list',
+  },
+  {
+    name: 'Console',
+    icon: <Cli />,
+    content: 'panes',
+  },
+  {
+    name: 'Configure',
+    icon: <Configure />,
+    content: 'panes',
+  },
+];
+
+export const MinimalSidebarExample = ({ mobile }) => {
+  const [activeItem, setActiveItem] = React.useState(1);
+
+  return (
+    <ResponsiveContext.Provider value={mobile && 'small'}>
+      <ResponsiveContext.Consumer>
+        {size => (
+          <AppContainer activeItem={activeItem}>
+            <Sidebar
+              gridArea="sidebar"
+              /* Sidebar should switch from column to row orientation 
+              when on smaller screens */
+              direction={size !== 'small' ? 'column' : 'row'}
+              /* Only display most critical navigation items in mobile 
+              contexts */
+              header={size !== 'small' && <SidebarHeader />}
+              footer={size !== 'small' && <SidebarFooter />}
+              background="blue!"
+              gap="xsmall"
```

Comment:
This gap between header and the rest of the icons looks like this is a lot smaller I believe it was 6px where the the other gaps was 24px. Wondering if there was a reason behind this ?

Reply:
Good catch. Fixing.